htdocs/ticket/index.php

```diff
@@ -376,7 +376,6 @@
 				$object->id = $objp->rowid;
 				$object->ref = $objp->ref;
 				$object->track_id = $objp->track_id;
-				$object->fk_statut = $objp->status;
```

Comment:
Attention, this should only be done when the DolDeprecatedHandler is integrated & when fk_statut is set as a deprecated property and mapped to status.

fk_status is not in the deprecatedProperties function in commonobject.class, but one can extend that method in a subclass.

```php
    protected function deprecatedProperties()
    {
        return ["fk_statut"=>"status"] + parent::deprecatedProperties();
    }
```

Maybe I should just add it to my branch.

Reply:
it is just for display tooltip

Reply:
Ok, that's why I do not touch this type of code as I do not have enough context 😉